pkg/driver/controller_server.go

```diff
@@ -408,7 +408,7 @@ func (driver *Driver) createVolume(
 	existingVolume, err := storageProvider.GetVolumeByName(name)
 	if err != nil {
 		log.Error("err: ", err.Error())
-		return nil, status.Error(codes.Unavailable, "Failed to check if volume exists")
+		return nil, status.Error(codes.Unavailable, fmt.Sprintf("Failed to check if volume exists. %s", err.Error()))
```

Comment:
nit: Do you want to log volume name as well? If its already captured in the error message, feel free to skip this comment.